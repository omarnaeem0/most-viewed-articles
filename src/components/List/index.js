import classNames from "classnames";
import styles from "./styles.module.scss";

export default function List({ children, className }) {
  return <ul className={classNames(styles.list, className)}>{children}</ul>;
}
