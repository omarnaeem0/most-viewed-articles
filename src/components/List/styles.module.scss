.list {
  max-width: 840px;
  padding: 0px 20px;
  margin: 0px auto;
  margin-bottom: 24px;
}
