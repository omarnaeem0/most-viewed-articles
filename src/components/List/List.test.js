import renderer from "react-test-renderer";
import List from ".";

test("List component", () => {
  const tree = renderer
    .create(
      <List>
        <li>listitem1</li>
        <li>listitem2</li>
      </List>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
