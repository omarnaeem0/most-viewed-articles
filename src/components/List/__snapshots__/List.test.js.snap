// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component 1`] = `
<ul
  className="list"
>
  <li>
    listitem1
  </li>
  <li>
    listitem2
  </li>
</ul>
`;
